### DISCUSSION TITLE
Proposal - Improve attribute parsing

### DISCUSSION THREAD

**Discussion by jmooring** [yuin/goldmark#278] [Ideas]
Users are accustomed to quoting HTML attributes, regardless of type, because many have no concept of type. And the browser doesn't care. I understand this is markdown, not HTML, but there is a similar expectation from non-developer users.

For example, when using goldmark-highlighting, quoting or not quoting an attribute makes a big difference.

Example|Notes
:--|:--
`{linenos="false"}`|Effectively setting it to true
`{hl_lines="[2]"}`|Ignored

Reference: <https://github.com/yuin/goldmark-highlighting/issues/28>

**Comment by yuin** [comment]
IMHO, I do not stand for this. I have been had many troubles related this kind of 'relaxed' binding.
